### ISSUE TITLE
POM file for org.openrefine.dependencies:secondstring:jar is invalid

### ISSUE THREAD

**Issue by thadguidry** [OpenRefine/OpenRefine#2473]
**Describe the bug**

WARNING error during compile that listed transitive dependency `Junit` is not provided.

**To Reproduce**
1. `refine build`
2. Alternatively can use `mvn dependency:tree -X`
3. See Current Results

**Current Results**

```
[DEBUG] =======================================================================
[WARNING] The POM for org.openrefine.dependencies:secondstring:jar:20100303 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for org.openrefine.dependencies:secondstring:20100303
[ERROR] 'dependencies.dependency.version' for junit:junit:jar is missing. @ 
```

The current POM on MavenCentral shows the scope of `junit` as `provided` ?

```xml
 <name>SecondString</name>
    <modelVersion>4.0.0</modelVersion>
    <description>This is a copy of the SecondString library from https://sourceforge.net/projects/secondstring/ to obtain a usable Maven artifact.</description>
    <dependencies>
        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <scope>provided</scope>
        </dependency>
    </dependencies>
```
**Expected behavior**
No warning.  The POM for `secondstring` should be made valid instead.
After which we probably should republish to MavenCentral.
Or fix it somehow else that I'm not intimately aware of at the moment.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Version: Firefox
 - JRE or JDK Version: JDK8

**OpenRefine (please complete the following information):**
 - Version: trunk (master)

**Comment by tfmorris** [comment]
Where is this POM? Is there a repo associated with our fork?

**Comment by thadguidry** [comment]
@tfmorris It's privately published by @wetneb from his laptop, who can tell you more.  I'd love to see a repo to hold the "openrefine.dependencies" or at least the Github Actions and POMs and scripts that pull it all together and do the publishing.  I.E.  Take what Antonin does manually now to maintain the "openrefine.dependencies" on his laptop and instead host it in a separate repo and productivize it as some GitHub CI/CD Actions.

**Comment by wetneb** [comment]
There is no particular repo, because there is no fork I am aware of: these jars are just the ones that were shipped in this repository when it was built with Ant and that I could not find an equivalent on Maven Central for.

There is nothing "private" about this publication process (or about "my laptop", what?), you can do the same by pushing the jars to Maven Central too. The changes I made were there to meet the minimal requirements from Maven Central in terms of metadata. There is no need for us to maintain these jars - we should just migrate away from them into something that is maintained, when the need arises.

@thadguidry there is no point about having a CI action on this repository to upload artifacts which, by definition, are not stored in this repository and are not meant to evolve anyway.

**Comment by thadguidry** [comment]
@wetneb ah thanks for providing more info.

**Comment by tfmorris** [comment]
I agree that we don't need an automatic publishing process. The pom.xml is the main artefact that is unique to us, but it sounds like we don't have those saved.

Switching to the official `secondstring` dependency is probably the right thing to do, but this isn't actually an OpenRefine dependency, but instead a transitive dependency from simile-vicino. While looking into this I realized that we never published the bug fix to vicino that we received, see if I can get the dependencies updated and publish a new version. I've opened #2959 for this.
